opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/mdc/MDCSpanDecorator.java

```diff
@@ -0,0 +1,76 @@
+package io.opentracing.contrib.mdc;
+
+import io.opentracing.Span;
+import io.opentracing.tag.StringTag;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.MDC;
+
+public class MDCSpanDecorator {
```

Comment:
How is this loaded?

Reply:
This library don't load any Decorator, it's left to the user to do so **or** to the SpringBoot library (PR is opened for that as well)